captura/views.py

```diff
@@ -224,7 +227,121 @@ def create_estudio(request):
     if request.method == 'POST':
         familia = Familia.objects.create()
         Estudio.objects.create(capturista=request.user.capturista, familia=familia)
-        return redirect(reverse('home'))
+        return redirect(reverse('captura:familia', kwargs={'id_familia': familia.id}))
+
+
+@login_required
+@user_passes_test(is_capturista)
+def familia(request, id_familia):
+    """ This view allows a capturista to capture the information related
+    to a specific family.
+    """
+    if request.method == 'POST':
+        instance = get_object_or_404(Familia, pk=id_familia)
+        form = FamiliaForm(request.POST, instance=instance)
+        if form.is_valid():
+            form.save()
+            return redirect(reverse('captura:integrantes',
+                                    kwargs={'id_familia': form.instance.pk}))
+    else:
+        familia = Familia.objects.get(pk=id_familia)
+        form = FamiliaForm(instance=familia)
+        form_view = 'captura/familia_form.html'
+        return render(request, 'captura/captura_base.html', {'form_view': form_view,
```

Comment:
Maybe use a context = {} dictionary like the view below to make it more readable? (Optional)